public/js/components/Header/Header.js

```diff
@@ -33,7 +34,7 @@ class Header extends React.Component {
               </Link>
             </header>
           : false}
-          {this.props.isFindPage ? <BrowseHeader /> : <EditHeader atom={this.props.atom} />}
+          {this.props.isFindPage ? <BrowseHeader /> : <EditHeader atom={this.props.atom} presence={this.props.presence}/>}
```

Comment:
Passing `presence` along to the Edit Header - so it can be used to switch off the errors found in there.